docs/contributing.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_position: 10
+title: "🍻 Contributing"
+---
+
+# 🍻 Contributing
+
+We welcome all contributions, bug reports, bug fixes, documentation improvements, enhancements, and ideas.
+
+## Getting Started
+The Chroma codebase is hosted on [Github](https://github.com/chroma-core/chroma) and issues are tracked on [Github Issues](https://github.com/chroma-core/chroma/issues). Please report any issues you find there making sure to fill out the correct [form for the type of issue you are reporting](https://github.com/chroma-core/chroma/issues/new/choose). In order to run Chroma locally you can follow the [Development Instructions](https://github.com/chroma-core/chroma/blob/main/DEVELOP.md).
+
+If you want to contribute and aren't sure where to get started you can search for issues with the [Good first issue](https://github.com/chroma-core/chroma/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) tag.
+
+The Chroma documentation (including this page!) is hosted on [Github](https://github.com/chroma-core/docs) as well. If you find any issues with the documentation please report them on the Github Issues page for the documentation [here](https://github.com/chroma-core/docs/issues). 
```

Comment:
the roadmap section also currently has a discussion of great areas for contribution